cbits/foundation_network.c

```diff
@@ -0,0 +1,16 @@
+#include "foundation_system.h"
+
+#if defined(FOUNDATION_SYSTEM_WINDOWS)
+# include <winsock2.h>
+#else
+# include "netdb.h"
+#endif
+
+int get_h_errno(void)
```

Comment:
if it's exposed it ought to be called `foundation_`.
also if WSAGetLastError, bindings to it directly (`sysWindowsSAGetLastError`) and preventing a needless export on windows sounds useful.